File: questions/1272. Remove Interval/remove.py
class Solution:
    def removeInterval(self, intervals: List[List[int]], toBeRemoved: List[int]) -> List[List[int]]:
        l = []
        s,e = toBeRemoved[0], toBeRemoved[1]
        for ele in intervals:
            if ele[0] >= e or ele[1] <= s:
                l.append(ele)
            else:
                if s > ele[0]:
                    l.append([ele[0], s])
                if e < ele[1]:
                    l.append([e, ele[1]])
        return l
        

File: questions/1272. Remove Interval/remove.cpp
class Solution {
public:
    vector<vector<int>> removeInterval(vector<vector<int>>& intervals, vector<int>& toBeRemoved) {
        vector<vector<int>> res;
        int s = toBeRemoved[0], e = toBeRemoved[1];
        for(auto& inter: intervals){
            if(inter[0] >= e || inter[1] <= s) res.push_back(inter);
            else{
                if(s > inter[0]){
                    res.push_back({inter[0], s});
                }
                if(e < inter[1]){
                    res.push_back({e, inter[1]});
                }
            }
        }
        return res;
    }
};
